portfolio/src/main/webapp/script.js

```diff
@@ -110,42 +115,53 @@ function createCommentElement(comment) {
   });
 
   commentElement.appendChild(timeElement);
-  commentElement.appendChild(titleElement);
+  commentElement.appendChild(messageElement);
   commentElement.appendChild(deleteButtonElement);
   return commentElement;
 }
 
-/** Tells the server to delete the task. */
+/** Tells the server to delete the task,
+where commentId is the id of the comment that needs to be deleted. */
 function deleteTask(commentId) {
   const params = new URLSearchParams();
   params.append('id', commentId);
   fetch('/delete-comment', {method: 'POST', body: params});
-  window.location.replace("index.html");
+  window.location.replace('index.html');
 }
 
-/**Onclicking the commentSettigns button show the dropdown-content */
+let SettingsToggleFlag = 0;
+/** Onclicking the commentSettigns button show the dropdown-content */
 function commentSettings() {
-  document.getElementById("dropdown-content").style.visibility = "visible"; 
-  document.getElementById("dropdown-content").style.display = "inline-block"; 
-}  
+  SettingsToggleFlag++;
+  if (SettingsToggleFlag % 2 == 0) {
+    document.getElementById('dropdown-content').style.display = 'none';
+  } else {
+    document.getElementById('dropdown-content').style.display = 'inline-block';
+  }
+}
 
-/**Reload along with posting the maxComments to url, when user selects the no. of comments he wants to see. */
+/** Reload along with posting the maxComments to url,
+when user selects the no. of comments he wants to see. */
 function sendForm() {
-  window.location.replace("index.html?maxComments="+ document.getElementById('quantity').value );
+  window.location.replace('index.html?maxComments='+
+  document.getElementById('quantity').value+'&languageCode='+
+  document.getElementById('language').value);
```

Comment:
This indentation is odd. The second and third line should be indented further than the first line.

Reply:
Oh, it was automatically done by the format code thing. I'll make sure to indent it the way you suggested.